### PR TITLE
[TD-12522]<enhence>(connector):csharp test cases update to follow sel…

### PR DESCRIPTION
**Pull request by xleili**
…f describe standard

### CODE REVIEW

File: src/connector/C#/src/TDengineDriver/TaosMultiBind.cs [hunk 1]
```diff
@@ -434,51 +434,48 @@ public static TAOS_MULTI_BIND MultiBindUBigInt(ulong?[] arr)
         }
         public static TAOS_MULTI_BIND MultiBindBinary(string[] arr)
         {
-            TAOS_MULTI_BIND multiBind = new TAOS_MULTI_BIND();
+             TAOS_MULTI_BIND multiBind = new TAOS_MULTI_BIND();
```
Comment (zitsen) [inline src/connector/C#/src/TDengineDriver/TaosMultiBind.cs]: Space?

File: src/connector/C#/src/test/FunctionTest/Utils.cs [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+using TDengineDriver;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Collections.Generic;
+namespace Test.UtilsTools
+{
+    public class UtilsTools
+    {
+
+        static string configDir = "/etc/taos";//"C:/TDengine/cfg";
```
Comment (zitsen) [inline src/connector/C#/src/test/FunctionTest/Utils.cs]: Make it clean or configurable.

File: src/connector/C#/src/test/XUnitTest/TestTDengineMeta.cs [hunk 3]
```diff
@@ -3,9 +3,14 @@
 using TDengineDriver;
 
 namespace TDengineDriver.Test
-{
+{
```
Comment (zitsen) [inline src/connector/C#/src/test/XUnitTest/TestTDengineMeta.cs]: Extra spaces not needed.

**Review by zitsen — CHANGES_REQUESTED** [review]
See my comments.

**Review by zitsen — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**